### ISSUE TITLE
Rulesets

### ISSUE THREAD

**Issue by AtomicFS** [9elements/bmc-test-go#2]
I have enabled ruleset protection for `main` branch, but since we do not have any CI/CD, it does not enforce `status check` yet. Once we have CI/CD such as linting, we should add it into the required `status check` list to prevent failing code to merge into `main`.

**Comment by pointbazaar** [comment]
Do we still need this issue? It looks like we already have linting enabled now

**Comment by pointbazaar** [comment]
closing due to outdated issue / unresponsiveness

**Comment by AtomicFS** [comment]
Yes, this is indeed fixed (not sure when or by whom). But as you can see in any PR there are some jobs with `Required` label, that is what we want.
